EP4497-CUSA16579_00-0000000000000001.xml

```diff
@@ -0,0 +1,103 @@
+<?xml version="1.0" encoding="utf-8"?>
+<pkgpatch>
+  <patchInfo description="" title_id="CUSA16579" content_id="EP4497-CUSA16579_00-0000000000000001" />
+  <patches>
+    <patch description="Duplicates from update 1.31 + Language English Only" pkg_required_version="01.00" update_version_required="01.31">
+      <file path="archive/ps4/content/basegame_1_engine.archive" />
+      <file path="archive/ps4/content/basegame_2_mainmenu.archive" />
+      <file path="archive/ps4/content/basegame_3_nightcity.archive" />
+      <file path="archive/ps4/content/basegame_3_nightcity_terrain.archive" />
+      <file path="archive/ps4/content/basegame_4_animation.archive" />
+      <file path="archive/ps4/content/basegame_4_appearance.archive" />
+      <file path="archive/ps4/content/basegame_4_gamedata.archive" />
+      <file path="archive/ps4/content/lang_ar_text.archive" />
+      <file path="archive/ps4/content/lang_cs_text.archive" />
+      <file path="archive/ps4/content/lang_en_text.archive" />
+      <file path="archive/ps4/content/lang_hu_text.archive" />
+      <file path="archive/ps4/content/lang_pl_text.archive" />
+      <file path="archive/ps4/content/lang_ru_text.archive" />
+      <file path="archive/ps4/content/lang_tr_text.archive" />
+      <file path="engine/config/base/general.ini" />
+      <file path="engine/shaderps4_final.cache" />
+      <file path="engine/staticshaderps4_final.cache" />
+      <file path="r6/cache/final.redscripts" />
+      <file path="r6/cache/tweakdb.bin" />
+      <file path="r6/config/settings/platform/orbis/options.json" />
+      <file path="r6/config/settings/options.json" />
+      <file path="r6/config/bumpersSettings.json" />
+      <file path="r6/config/inputContexts.xml" />
+      <file path="r6/config/inputDeadzones.xml" />
+      <file path="r6/config/inputUserMappings.xml" />
+      <file path="r6/config/uiInputActions.xml" />
+      <file path="archive/ps4/content/lang_pl_voice.archive" />

```

Comment:
I prefer to have a patch only removing duplicated. Then It could be better to have one patch for each language.
 <patch description="Language English Only" pkg_required_version="01.00" update_version_required="01.00">
      <file path="archive/ps4/content/lang_pl_voice.archive" />
       <file path="archive/ps4/content/lang_ru_voice.archive" />
    </patch>
     <patch description="Language Russian Only" pkg_required_version="01.00" update_version_required="01.00">
      <file path="archive/ps4/content/lang_pl_voice.archive" />
       <file path="archive/ps4/content/lang_en_voice.archive" />
    </patch>
     <patch description="Language Polish Only" pkg_required_version="01.00" update_version_required="01.00">
      <file path="archive/ps4/content/lang_ru_voice.archive" />
       <file path="archive/ps4/content/lang_en_voice.archive" />
    </patch>
    
And then one patch for each language of the update (I don't know if the update replaces the voice files)
<patch description="Language English Only" pkg_required_version="01.31" update_version_required="01.31">
      <file path="archive/ps4/content/lang_pl_voice.archive" />
       <file path="archive/ps4/content/lang_ru_voice.archive" />
    </patch>
     <patch description="Language Russian Only" pkg_required_version="01.31" update_version_required="01.31">
      <file path="archive/ps4/content/lang_pl_voice.archive" />
       <file path="archive/ps4/content/lang_en_voice.archive" />
    </patch>
     <patch description="Language Polish Only" pkg_required_version="01.31" update_version_required="01.31">
      <file path="archive/ps4/content/lang_ru_voice.archive" />
       <file path="archive/ps4/content/lang_en_voice.archive" />
    </patch>

By this way we have separate patches for duplicated content and laguages, it can be applied both at the same time or separatelly